ui/src/views/Connectors.vue

```diff
@@ -704,7 +705,7 @@ export default {
         Password: connector.props.Password,
         Retriever: connector.props.Retriever,
         Username: connector.props.Username,
-        FilterCheck: connector.props.FilterCheck ? "enabled" : "disabled",
+        FilterCheck: connector.props.FilterCheck == "enabled" ? "enabled" : "disabled",
```

Comment:
You can read the `FilterCheck` value without manipulate it, leave only:
```
FilterCheck: connector.props.FilterCheck,
```

Reply:
Changed.